### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ternera**
I cleaned up the grammar and punctuation in the README file. This makes things easier to read and adds consistency.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -271,7 +271,7 @@ Visit [http://localhost:3000/](http://localhost:3000/) to view your running vers
 
 As Access is a web application packaged with Docker, it can easily be deployed to a Kubernetes cluster. We've included example Kubernetes yaml objects you can use to deploy Access in the [examples/kubernetes](https://github.com/discord/access/tree/main/examples/kubernetes) directory.
 
-These examples include a Deployment, Service, Namespace, Service Account object for serving the stateless web application. Additionally there are examples for deploying the `flask sync` and `flask notify` commands as cronjobs to periodically synchronize users, groups, and their memberships and send expiring access notifications respectively.
+These examples include a Deployment, Service, Namespace and Service Account object for serving the stateless web application. Additionally there are examples for deploying the `flask sync` and `flask notify` commands as cronjobs to periodically synchronize users, groups, and their memberships and send expiring access notifications respectively.
```
Comment (somethingnew2-0) [inline README.md:274]: Include the Oxford comma

```suggestion
These examples include a Deployment, Service, Namespace, and Service Account object for serving the stateless web application. Additionally there are examples for deploying the `flask sync` and `flask notify` commands as cronjobs to periodically synchronize users, groups, and their memberships and send expiring access notifications respectively.
```

File: README.md [hunk 2]
```diff
@@ -24,20 +24,20 @@ The access service exists to help answer the following questions for each person
...
 
 ## Development Setup
 
-Access is a React+Typescript single-page application (SPA) with a Flask API that connects to the Okta API
+Access is a React and Typescript single-page application (SPA) with a Flask API that connects to the Okta API.
 
 You'll need an Okta API Token from an Okta user with the `Group Admin` and `Application Admin`
 Okta administrator roles granted as well as all Group permissions (ie. `Manage groups` checkbox checked)
-in a custom Admin role. If you want to manage Groups which grant Okta Admin permissions, then the Okta API
+in a custom Admin role. If you want to manage Groups which grant `Okta Admin` permissions, then the Okta API
```
Comment (somethingnew2-0) [inline README.md:35]: Okta Admin shouldn't be highlighted as it isn't referencing an Okta administrator role

```suggestion
in a custom Admin role. If you want to manage Groups which grant Okta administrator permissions, then the Okta API
```

**Comment by ternera** [comment]
Looks good. Thanks for catching those @somethingnew2-0!

File: README.md [hunk 3]
```diff
@@ -24,20 +24,20 @@ The access service exists to help answer the following questions for each person
   - Who has access to my application?
   - How do I setup access for a new application?
   - How do I create a new access group for my application?
-  - How do I give access to a role to one of my application's groups?
+  - How do I give one of my application's groups access to a role?
```
Comment (eguerrant) [inline README.md:27]: Should be "How do I give a role access to one of my application's groups?"

Comment (ternera) [reply README.md]: Fixed it!

**Review by somethingnew2-0 — APPROVED** [review]

**Pull request merged**